wpiutil/build.gradle

```diff
@@ -183,6 +183,9 @@ cppHeadersZip {
     from('src/main/native/thirdparty/expected/include') {
         into '/'
     }
+    from('src/main/native/include/thirdparty/concurrentqueue/include') {
+        into '/'
+    }
```

Comment:
To keep sort order, put this before expected.

Reply:
yeah, c does go before e doesn't it...